packages/react/ds/src/header/header.tsx

```diff
@@ -6,6 +6,7 @@ import HeaderSearch from './components/header-search.js';
 
 export type HeaderProps = {
   logoLink: string;
+  search_url: string;
```

Comment:
I will set this optional or have another flag to define whether the search is active.
In some cases, the search will not be part of the application, and we will need to hide the button.

Reply:
good idea. I will put this as optional and if the user doesn't use this prop, the search component will not render